scripts/build_utils.sh

```diff
@@ -651,6 +651,8 @@ use_ppa() {
                     use_ppa=true;;
                 bionic)
                     use_ppa=false;;
+                focal)
+                    use_ppa=true;;
```

Comment:
just need to make sure that https://tracker.ceph.com/issues/50218 does not apply to focal.

Reply:
good to know; if that shows up, it will be immediately obvious from teuthology testing